pysidt/sidt.py

```diff
@@ -339,6 +339,23 @@ def read_nodes(file):
 
 
 class MultiEvalSubgraphIsomorphicDecisionTree(SubgraphIsomorphicDecisionTree):
+    """
+    Makes prediction for a molecule based on multiple evaluations.
+
+    Args:
+        `decomposition`: method to decompose a molecule into groups.
+        `root_group`: root group for the tree
+        `nodes`: dictionary of nodes for the tree
+        `n_splits`: number of splits to try for each node. Default is 1.
+        `iter_max`: maximum number of iterations for each split. Default is 2.
```

Comment:
inter_max: maximum number of times the extension generation algorithm is allowed to expand structures looking for additional splits
iter_item_cap: the maximum number of structures the extension generation algorithm can send for expansion